js/battle.js

```diff
@@ -5331,6 +5331,7 @@ var Battle = (function () {
 		case 'c':
 			name = args[1];
 			if (this.ignoreSpects && (name.charAt(0) === ' ' || name.charAt(0) === '+')) break;
+			if (this.ignoreOpponent && this.getSide(name) === this.yourSide) break;
```

Comment:
I thought about doing that, but it would result in users being unable to see their own messages, like using /ignore on yourself. It would probably be confusing if that happened instead of only ignoring the opponent's messages.